### PR TITLE
Bump up artifact engine version

### PR DESCRIPTION
**Pull request by gykuma**

### CODE REVIEW

File: Tasks/DownloadFileshareArtifactsV0/task.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 1,
-    "Minor": 139,
```
Comment (omeshp) [inline Tasks/DownloadFileshareArtifactsV0/task.json]: update task.loc.json also?

File: Tasks/JenkinsDownloadArtifactsV1/task.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "demands": [],
     "version": {
         "Major": 1,
-        "Minor": 138,
```
Comment (omeshp) [inline Tasks/JenkinsDownloadArtifactsV1/task.json]: update task.loc.json also?

File: Tasks/DownloadFileshareArtifactsV0/task.json (hunk 1 above)
Comment (gykuma) [reply Tasks/DownloadFileshareArtifactsV0/task.json]: updated

File: Tasks/JenkinsDownloadArtifactsV1/task.json (hunk 2 above)
Comment (gykuma) [reply Tasks/JenkinsDownloadArtifactsV1/task.json]: updated

**Pull request merged**